build.sbt

```diff
@@ -22,9 +29,13 @@ libraryDependencies ++= Seq(
   "org.scalatest" %% "scalatest" % "2.1.7" % "test",
   "com.typesafe.akka" %% "akka-cluster" % akkaVersion,
   "com.typesafe.akka" %% "akka-contrib" % akkaVersion,
-  "org.reactivemongo" %% "reactivemongo" % "0.10.5.0.akka23"
+  "org.reactivemongo" %% "reactivemongo" % "0.10.5.0.akka23",
+  "org.apache.spark" %% "spark-core" % "1.3.0" % "provided",
```

Comment:
It would be better if the main lookup project wouldn't depend on Spark (for example if we deploy on Azure we don't want it to fetch Spark). Can we find a way to make it two separate SBT projects?